### DISCUSSION TITLE
Create project in a single command

### DISCUSSION THREAD

**Discussion by Sleon4** [alex8088/electron-vite#428] [General]
Good day,

I want to ask you if it is possible to create an electron-vite project in a command directly, for example:

```bash
npm create @quick-start/electron my-app -- --template react --type typescript --electron-updater no --mirror-proxy no
```

I would like to have the ability to create projects more quickly in a single command, or is there already a configuration that does this?

**Comment by alex8088** [comment]
```sh
npm create @quick-start/electron my-app -- --template react-ts --skip
npm create @quick-start/electron my-app -- --template vue --skip
```

**Reply by Sleon4** [reply]
@alex8088 You are the best friend, success in your life <3
